From fd38b763a2d0af061b94301589004d09ea12ff46 Mon Sep 17 00:00:00 2001
From: Joey Riches <josephriches@gmail.com>
Date: Fri, 29 Sep 2023 23:38:17 +0100
Subject: [PATCH 1/1] quirks: Disable quirk removing two-finger scrolling for
 HP Pavillion

Opiniated: Personally affected by this and I would much rather live with
any potential bugginess than be forced to use edge-scrolling. I can't
even replicate the original bug.

Upstream bug: https://bugs.freedesktop.org/show_bug.cgi?id=91135
---
 quirks/50-system-hp.quirks | 11 +++++++----
 1 file changed, 7 insertions(+), 4 deletions(-)

diff --git a/quirks/50-system-hp.quirks b/quirks/50-system-hp.quirks
index 80e7b9b4..6b959141 100644
--- a/quirks/50-system-hp.quirks
+++ b/quirks/50-system-hp.quirks
@@ -14,10 +14,13 @@ MatchName=*SynPS/2 Synaptics TouchPad
 MatchDMIModalias=dmi:*svnHewlett-Packard:*pnHPCompaq8510w*
 AttrEventCode=-BTN_TOOL_DOUBLETAP;-BTN_TOOL_TRIPLETAP;

-[HP Pavilion dmi4]
-MatchName=*SynPS/2 Synaptics TouchPad
-MatchDMIModalias=dmi:*svnHewlett-Packard:*pnHPPaviliondm4NotebookPC*
-ModelHPPavilionDM4Touchpad=1
+# SOLUS: Downstream change
+# We will live with any potential bugginess rather than force
+# edge scrolling here.
+# [HP Pavilion dmi4]
+# MatchName=*SynPS/2 Synaptics TouchPad
+# MatchDMIModalias=dmi:*svnHewlett-Packard:*pnHPPaviliondm4NotebookPC*
+# ModelHPPavilionDM4Touchpad=1

 # Touchpad is a clickpad but INPUT_PROP_BUTTONPAD is not set, see
 # https://bugs.freedesktop.org/show_bug.cgi?id=97147
--
2.42.0
